### ISSUE TITLE
`usage` is broken

### ISSUE THREAD

**Issue by Domiii** [vitalets/bro-fs#7]
## Problem
[`fs.usage`](https://github.com/vitalets/bro-fs/blob/master/src/quota.js#L16) is not working properly: `grantedBytes` will always be `undefined`.

That is because it is calling `queryUsageAndQuota` under the hood which accepts a callback with two arguments (as explained [here](https://stackoverflow.com/a/29662985)).

## Solution
For this to work, [`promiseCall`](https://github.com/vitalets/bro-fs/blob/df345b7122ce68fd38eca20bcca8797b7f335da0/src/utils.js#L5) must merge the resulting arguments into a single argument before calling `resolve`. That is because promises can never yield/return/propagate (or whatever you want to call it) more than a single value, [as explained here](https://stackoverflow.com/a/28703648).

A (somewhat unoptimized) solution would be something along the lines of:

```
const fullArgs = args.concat([(...results) => resolve(results), errback]);
```

(which then also needs to be handled properly in [`usage`](https://github.com/vitalets/bro-fs/blob/master/src/quota.js#L16))

instead of the current version:

```
const fullArgs = args.concat([resolve, errback]);
```

My current work-around is to work directly with the underlying API, until the problem is fixed, e.g.:

```
navigator.webkitPersistentStorage.queryUsageAndQuota((usedBytes, grantedBytes) => {
    // ...
});
```

**Comment by vitalets** [comment]
@Domiii thanks for reporting! I will have a look on that!

**Comment by vitalets** [comment]
Released as 0.4.0.
